syntax = "proto3";

package lamp;

service LampService {
  rpc ligarLampada (LigarLampadaRequest) returns (Status);
  rpc desligarLampada (DesligarLampadaRequest) returns (Status);
}

message LigarLampadaRequest {
  // Campos do pedido para ligar a l칙mpada, se necess치rio
}

message DesligarLampadaRequest {
  // Campos do pedido para desligar a l칙mpada, se necess치rio
}

message Status {
  string message = 1;
}
